extension/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 -->
 
+## [1.6.0] -
+
+- New features:
+  - Trans-units with empty sources are now flagged. An empty source is often not supposed to be translated and the caption should be given the attribute `Locked = true`. When running `NAB: Refresh XLF files from g.xlf` a note is added to the trans-unit suggesting this: `Source contains only white-space, consider using 'Locked = true' to avoid translation of unnecessary texts`. Suggested in ([issue 179](https://github.com/jwikman/nab-al-tools/issues/179)).
```

Comment:
Remove one of the parenthesis?
```suggestion
  - Trans-units with empty sources are now flagged. An empty source is often not supposed to be translated and the caption should be given the attribute `Locked = true`. When running `NAB: Refresh XLF files from g.xlf` a note is added to the trans-unit suggesting this: `Source contains only white-space, consider using 'Locked = true' to avoid translation of unnecessary texts`. Suggested in [issue 179](https://github.com/jwikman/nab-al-tools/issues/179).
```